### PR TITLE
IDs: replace/remove "unknown" and "regular"

### PR DESCRIPTION
**Pull request by patstirling**
**Is this a new addition or an update to an existing factor?**

- [ ] New
- [x] Update

**What category or categories does it affect?**

All categories

**What?**

Replaced "average" with "unknown", or removed it where unnecessary ("regular" in one case). Made changes to ID and some descriptions to make sure context was clear. Spotted some esoteric issues and resolved.

**Why?**

The OEFDB uses "unknown" (soon "na") to reflect where a given aspect of an emission factor is either not provided or is provided as an average (e.g. where the type of fuel a car uses is not provided, "fuel_source_unknown" rather than "fuel_source_average")

### CODE REVIEW

File: OpenEmissionFactorsDB.csv [hunk 1]
```diff
@@ -12,36 +12,36 @@ Transport,Air Travel,passenger_flight-route_type_unknown-aircraft_type_unknown-d
...
+Transport,Vehicle,passenger_vehicle-vehicle_type_car-fuel_source_unknown-engine_size_unknown-vehicle_age_unknown-vehicle_weight_unknown,Car (average),kg-CO2e/km,0.17148,,BEIS,2021,GB,2021/07/23,Emission intensity of average car. Unknown engine size. No upstream or downstream LCA activity is included. Available in BEIS Greenhouse Gas Reporting: Conversion Factors 2021.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021,fuel_combustion
+Transport,Vehicle,passenger_vehicle-vehicle_type_car-fuel_source_unknown-engine_size_unknown-vehicle_age_unknown-vehicle_weight_unknown,CNG car (average),kg-CO2e/km,0.17624,,BEIS,2021,GB,2021/07/23,Emission intensity of average CNG car. A compressed version of the same natural gas you receive in the home. 2.0-litre engine +. When compressed can be used as an alternative vehicle fuel. Unknown engine size. No upstream or downstream LCA activity is included. Available in BEIS Greenhouse Gas Reporting: Conversion Factors 2021.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021,fuel_combustion
```
Comment (AnastasiaLobanova) [inline OpenEmissionFactorsDB.csv:16]: Here cng turned into unknown

File: OpenEmissionFactorsDB.csv [hunk 2]
```diff
@@ -12,36 +12,36 @@ Transport,Air Travel,passenger_flight-route_type_unknown-aircraft_type_unknown-d
...
-Transport,Vehicle,passenger_vehicle-vehicle_type_motorcycle_more_than_250cc_average-fuel_source_unknown-engine_size_unknown-vehicle_age_unknown-vehicle_weight_unknown,Motorcycle (>250cc),kg-CO2e/km,0.165,60,ADEME,2018,FR,2021/08/20,Emission intensity of average motorcycle > 250 cc. Updated April 2020. Published by ADEME. Provided by the HBEFA & Yearly Compte des Transports.,https://www.bilans-ges.ademe.fr/en/accueil,fuel_upstream_and_fuel_combustion
+Transport,Vehicle,passenger_vehicle-vehicle_type_car-fuel_source_hev-engine_size_unknown-vehicle_age_unknown-vehicle_weight_unknown,Motorcycle (>250cc),kg-CO2e/km,0.165,60,ADEME,2018,FR,2021/08/20,Emission intensity of average motorcycle > 250 cc. Updated April 2020. Published by ADEME. Provided by the HBEFA & Yearly Compte des Transports.,https://www.bilans-ges.ademe.fr/en/accueil,fuel_upstream_and_fuel_combustion
```
Comment (AnastasiaLobanova) [inline OpenEmissionFactorsDB.csv:26]: the name says it a motorcycle but was changed to a car. Also the fuel source was replaced

File: OpenEmissionFactorsDB.csv [hunk 3]
```diff
@@ -12,36 +12,36 @@ Transport,Air Travel,passenger_flight-route_type_unknown-aircraft_type_unknown-d
...
-Transport,Road Travel,passenger_vehicle-vehicle_type_car-fuel_source_E85-distance_average-engine_size_unknown,E85 passenger car average distance,kg-CO2e/pkm,0.0754,60,ADEME,2018,FR,2021/08/20,Emission intensity of E85 passenger car average distance. Updated April 2020. Published by ADEME. Provided by the HBEFA & Yearly Compte des Transports.,https://www.bilans-ges.ademe.fr/en/accueil,fuel_upstream_and_fuel_combustion
+Transport,Road Travel,passenger_vehicle-vehicle_type_car-fuel_source_E85-distance_unknown-engine_size_unknown,E85 passenger car average distance,kg-CO2e/pkm,0.0754,60,ADEME,2018,FR,2021/08/20,Emission intensity of E85 passenger car average distance. Updated April 2020. Published by ADEME. Provided by the HBEFA & Yearly Compte des Transports.,https://www.bilans-ges.ademe.fr/en/accueil,fuel_upstream_and_fuel_combustion
```
Comment (AnastasiaLobanova) [inline OpenEmissionFactorsDB.csv:32]: Uuu! E85 in the capital letters should be changed to small letters! I remembered we had it somewhere but forgot to change it afterward

File: OpenEmissionFactorsDB.csv [hunk 4]
```diff
@@ -1000,7 +1000,7 @@ Transport,Road Transport,freight_vehicle-vehicle_type_truck_medium_or_heavy-fuel
...
-Transport,Rail Travel,passenger_train-route_type_urban-fuel_source_average,National average for urban passenger train,kg-CO2e/pkm,0.03772874,,MfE,2020,NZ,2021/09/10,Emission intensity for urban passenger train (national average) including fuel consumption. Public transport passenger travel emission factors. Published by the Ministry for the Environment of New Zealand in 2020.,https://environment.govt.nz/publications/measuring-emissions-detailed-guide-2020/,fuel_consumption
+Transport,Rail Travel,passenger_vehicle-vehicle_type_local_bus-fuel_source_unknown-distance_unknown-engine_size_unknown,National average for urban passenger train,kg-CO2e/pkm,0.03772874,,MfE,2020,NZ,2021/09/10,Emission intensity for urban passenger train (national average) including fuel consumption. Public transport passenger travel emission factors. Published by the Ministry for the Environment of New Zealand in 2020.,https://environment.govt.nz/publications/measuring-emissions-detailed-guide-2020/,fuel_consumption
```
Comment (AnastasiaLobanova) [inline OpenEmissionFactorsDB.csv:1003]: Here the replacement of the ID is wrong

File: OpenEmissionFactorsDB.csv [hunk 5]
```diff
@@ -1030,7 +1030,7 @@ Transport,Vehicle,commercial_vehicle-vehicle_type_hgv-fuel_source_diesel-engine_
...
-Transport,Road Travel,passenger_vehicle-vehicle_type_bus-fuel_source_average-distance_unknown-engine_size_unknown,National average bus,kg-CO2e/pkm,0.136,,MfE,2020,NZ,2021/09/10,Emission intensity of national average bus. Provided by the Ministry for the Environment. Published by the Ministry for the Environment of New Zealand in 2020.,https://environment.govt.nz/publications/measuring-emissions-detailed-guide-2020/,fuel_combustion
+Transport,Road Travel,passenger_vehicle-vehicle_type_car-fuel_source_phev_electricity_consumption-engine_size_unknown-vehicle_age_unknown-vehicle_weight_unknown,National average bus,kg-CO2e/pkm,0.136,,MfE,2020,NZ,2021/09/10,Emission intensity of national average bus. Provided by the Ministry for the Environment. Published by the Ministry for the Environment of New Zealand in 2020.,https://environment.govt.nz/publications/measuring-emissions-detailed-guide-2020/,fuel_combustion
```
Comment (AnastasiaLobanova) [inline OpenEmissionFactorsDB.csv:1033]: I think the ID from line 1003 should be here?

**Review by AnastasiaLobanova — CHANGES_REQUESTED** [review]

File: OpenEmissionFactorsDB.csv (hunk 3 above)
Comment (patstirling) [reply OpenEmissionFactorsDB.csv:32]: leaving this for a later PR (for workflow reasons)

File: OpenEmissionFactorsDB.csv (hunk 4 above)
Comment (patstirling) [reply OpenEmissionFactorsDB.csv]: resolved

File: OpenEmissionFactorsDB.csv (hunk 5 above)
Comment (patstirling) [reply OpenEmissionFactorsDB.csv]: resolved

File: OpenEmissionFactorsDB.csv [hunk 6]
```diff
@@ -12,36 +12,36 @@ Transport,Air Travel,passenger_flight-route_type_unknown-aircraft_type_unknown-d
...
-Transport,Vehicle,passenger_vehicle-vehicle_type_average_car-fuel_source_lpg-engine_size_unknown-vehicle_age_unknown-vehicle_weight_unknown,LPG car (average),kg-CO2e/km,0.19828,,BEIS,2021,GB,2021/07/23,Emission intensity of average LPG car. Unknown engine size. No upstream or downstream LCA activity is included. Available in BEIS Greenhouse Gas Reporting: Conversion Factors 2021.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021,fuel_combustion
+Transport,Road Travel,passenger_vehicle-vehicle_type_car-fuel_source_diesel-engine_size_unknown-vehicle_age_unknown-vehicle_weight_unknown,Average passenger car short distance,kg-CO2e/pkm,0.111,60,ADEME,2018,FR,2021/08/20,Emission intensity of average passenger car short distance. Updated April 2020. Published by ADEME. Provided by the HBEFA & Yearly Compte des Transports.,https://www.bilans-ges.ademe.fr/en/accueil,unknown
```
Comment (AnastasiaLobanova) [inline OpenEmissionFactorsDB.csv:19]: here fuel-source is diesel suddently

**Review by maryarbab — DISMISSED** [review]

File: OpenEmissionFactorsDB.csv [hunk 7]
```diff
@@ -981,7 +981,7 @@ Transport,Road Transport,freight_vehicle-vehicle_type_light_delivery-fuel_source
...
-Transport,Road Travel,passenger_vehicle-vehicle_type_local_bus-fuel_source_average-distance_unknown-engine_size_unknown,Local London Bus,kg-CO2e/pkm,0.07718,,BEIS,2021,GB-LON,2021/07/23,Emission intensity of local London bus. Available in BEIS Greenhouse Gas Reporting: Conversion Factors 2021.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021,fuel_combustion
+Transport,Road Travel,passenger_vehicle-vehicle_type_local_bus-fuel_source-distance_unknown-engine_size_unknown,Local London Bus,kg-CO2e/pkm,0.07718,,BEIS,2021,GB-LON,2021/07/23,Emission intensity of local London bus. Available in BEIS Greenhouse Gas Reporting: Conversion Factors 2021.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021,fuel_combustion
```
Comment (AnastasiaLobanova) [inline OpenEmissionFactorsDB.csv:984]: unknown is missing here after the fuel_type

**Review by AnastasiaLobanova — CHANGES_REQUESTED** [review]

File: OpenEmissionFactorsDB.csv (hunk 6 above)
Comment (patstirling) [reply OpenEmissionFactorsDB.csv]: done

File: OpenEmissionFactorsDB.csv (hunk 7 above)
Comment (patstirling) [reply OpenEmissionFactorsDB.csv]: done - thanks!

**Review by maryarbab — DISMISSED** [review]

**Review by AnastasiaLobanova — APPROVED** [review]

**Review by maryarbab — APPROVED** [review]

**Pull request merged**